modules/contact-form/grunion-contact-form.php

```diff
@@ -232,6 +232,97 @@ function __construct() {
 		 */
 		wp_register_style( 'grunion.css', GRUNION_PLUGIN_URL . 'css/grunion.css', array(), JETPACK__VERSION );
 		wp_style_add_data( 'grunion.css', 'rtl', 'replace' );
+
+		if ( function_exists( 'register_block_type' ) ) {
```

Comment:
Should we use `Jetpack_Gutenberg::is_gutenberg_available` here?

Reply:
Yes good call. The plan is to convert things to jetpack_register_block calls which do that check as well.